services/apimap-sender.js

```diff
@@ -1,12 +1,15 @@
 'use strict';
 var request = require('superagent');
 var ServiceUrlGetter = require('./service-url-getter');
+var ApimapSorter = require('./apimap-sorter');
 var logger = require('./logger');
 
 function ApimapSender(envSecret, apimap) {
   this.perform = function() {
     var urlService = new ServiceUrlGetter().perform();
 
+    apimap = ApimapSorter(apimap);
+
     request
       .post(urlService + '/forest/apimaps')
       .send(apimap)
```

Comment:
We want to ensure that the apimap sent in the body (as a String) is formatted like this:

```javascript
// "data", "included", "meta" alphabetical order
{
  // "type", "id" order for the "data" array
  "data": [{ // "type", "id", "attributes" order (non-alphabetical to simplify the readability)
    "type": "collections",
    "id": "A",
    "attributes": { // "name", <other keys in alphabetical order>, "fields" order (non-alphabetical to simplify the readability)
      "name": "A",
      "is-searchable": true,
      "fields": [{ // "field", "type", <other keys in alphabetical order> order (non-alphabetical to simplify the readability)
        "field": "A",
        "type": "Number"
      }, {
        "field": "B",
        "type": "String"
      }]
    }
  }, {
    "type": "collections",
    "id": "B",
    "attributes": {
      "name": "B",
      "display-name": "B",
      "icon": "...",
      "integration": "...",
      "name-old": "...",
      "only-for-relationships": true,
      "pagination-type": "...",
      "is-read-only": true,
      "is-virtual": true,
      "fields": [{
        "field": "A",
        "type": "String",
        "is-filterable": false
      }, {
        "field": "B",
        "type": "Boolean",
        "is-filterable": false
      }],
    }
  },
  // "type", "id" order for the "included" array
  "included": [{ // "type", "id", "attributes" order (non-alphabetical to simplify the readability)
    "type": "actions",
    "id": "A.D",
    "attributes": { // "name", "fields" alphabetical order
      "name": "D",
      "fields": [{ // "field", "type", <other keys in alphabetical order> order (non-alphabetical to simplify the readability)
        "field": "D",
        "type": "...",
        "defaultValue": "..."
        "description": "...",
        "enums": ["...", "..."],
        "isRequired": true,
      }]
    }
  }, {
    "type": "segments",
    "id": "A.C",
    "attributes": { // "name", <other keys in alphabetical order> order (non-alphabetical to simplify the readability)
      "name": "C"
    }
  }, {
    "type": "segments",
    "id": "B.E",
    "attributes": {
      "name": "E"
    }
  }]
  "meta": { // "database_type", "liana", "liana_version", "orm_version" alphabetical order
    "database_type": "...",
    "liana": "...",
    "liana_version": "...",
    "orm_version": "..."
  }
}
```

And we need at least one test on this change.